src/frontend-proxy/envoy.tmpl.yaml

```diff
@@ -197,6 +197,7 @@ static_resources:
     - name: flagd-ui
       type: STRICT_DNS
       lb_policy: ROUND_ROBIN
+      timeout: 0s
```

Comment:
When trying to run the demo after this change, I got this error:

```
Protobuf message (type envoy.config.bootstrap.v3.Bootstrap reason INVALID_ARGUMENT: invalid JSON in envoy.config.bootstrap.v3.Bootstrap @ static_resources.clusters[5]: message envoy.config.cluster.v3.Cluster, near 1:5969 (offset 5968):  no   such field:   'timeout') has unknown fields
```

Frontend-Proxy didn't run with this.